internal/coreinternal/processor/filterconfig/config.go

```diff
@@ -142,23 +161,32 @@ func (mp *MatchProperties) ValidateForSpans() error {
 	}
 
 	if len(mp.Services) == 0 && len(mp.SpanNames) == 0 && len(mp.Attributes) == 0 &&
-		len(mp.Libraries) == 0 && len(mp.Resources) == 0 {
-		return errors.New(`at least one of "services", "span_names", "attributes", "libraries" or "resources" field must be specified`)
+		len(mp.Libraries) == 0 && len(mp.Resources) == 0 && len(mp.SpanKinds) == 0 {
+		return ErrMissingRequiredField
+	}
+
+	if len(mp.SpanKinds) > 0 && mp.MatchType == "strict" {
+		for _, kind := range mp.SpanKinds {
+			if !spanKinds[kind] {
+				return errors.New("span_kinds string must match one of the standard span kinds when match_type=strict ")
```

Comment:
nit: If we print the list of valid values it would be clearer for people what values they can write
```suggestion
                               // TODO: loop over `spanKinds` and create a `validSpanKinds` slice
				return fmt.Errorf("span_kinds string must match one of the standard span kinds when match_type=strict: %v", validSpanKinds)
```

Reply:
yes absolutely. I need to add a test.